README.md

```diff
@@ -91,3 +91,8 @@ Questions & Comments
 --------------------
 
 For any and all feedback, please use the Issues on this repository. 
+
+License
+--------------------
+
+By downloading the .zip you are agreeing to the terms in the project [EULA](https://github.com/dotnet/cli/blob/master/packaging/osx/sharedframework/resources/en.lproj/eula.rtf).
```

Comment:
Will do, but is this EULA going to be in ```master``` when its released or should I point directly to the ```rel/1.0.0 branch``` version?